portal/templates/contract/components/loan-transactions-table.njk

```diff
@@ -67,16 +68,15 @@
 
                   {% set componentData = {user: params.user, deal: params.deal, editable: params.editable, facility: loan, facilityName: 'loan', hasConfirmedCoverStartDate: hasConfirmedCoverStartDate, facilityTableIndex: loop.index} %}
 
-                  {% set userIsMaker = params.user.roles.includes('maker')%}
-
                   {% set dealSubmissionType = params.deal.submissionType %}
 
+                  {% set userIsMaker = params.user.roles.includes('maker')%}
+
                   {% set requestedCoverDateCanEdit = userIsMaker and loan.status !== "Maker's input required" and dealStatusAllowsRequestedCoverDateChange and dealSubmissionType === 'Manual Inclusion Application' and loan.facilityStage === 'Unconditional' and loan.issueFacilityDetailsSubmitted %}
```

Comment:
I know this has not been included in your PR but can this also please be feed from the controller?

Reply:
I think this issue is bigger than this ticket -- we have a lot of complex values being calculated inside nunjucks components -- I'd like to have a think about the best way to capture these (I'm sure there's a lot of overlap with these too).